python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -80,7 +80,9 @@ def test_read_json_nested(tmp_path):
         }
     )
     kwargs = dict(orient="records", lines=True)
-    with tmp_path / "data.json" as f:
+    with tmp_path / "data.json" as f, dask.config.set(
+        {"dataframe.convert-string": False}
+    ):
```

Comment:
Just a note that there is nothing we can do in dask-cudf to avoid the need for this. This is just one of several instances where https://github.com/dask/dask/issues/10631 is a problem (dask is generally too aggressive in converting "object" dtypes to "string")